components/brave_wallet_ui/page/screens/swap/android/swap.tsx

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+
+import { initLocale } from 'brave-ui'
+import { NetworkInfo, Swap as SwapInterface } from '@brave/swap-interface'
+import '@brave/swap-interface/dist/style.css'
+
+// Utils
+import { getLocale } from '$web-common/locale'
+import { loadTimeData } from '../../../../../common/loadTimeData'
+
+// Selectors
+import { WalletSelectors } from '../../../../common/selectors'
+import {
+  useSafeWalletSelector,
+  useUnsafeWalletSelector
+} from '../../../../common/hooks/use-safe-selector'
+
+// actions
+import * as WalletActions from '../../../../common/actions/wallet_actions'
+
+// Hooks
+import { useLib } from '../../../../common/hooks'
+
+// Types
+import { BraveWallet, WalletAccountType } from '../../../../constants/types'
+
+// Components
+import { store } from '../../../store'
+
+// Adapters
+import {
+  makeBlockchainToken,
+  makeNetworkInfo,
+  makeWalletAccount,
+  makeSwapService,
+  makeSOLSendTransaction,
+  makeGetGasPrice,
+  makeGetGasPrice1559,
+  makeGetERC20ApproveData,
+  makeETHSendTransaction,
+  makeSwitchNetwork,
+  makeSwitchAccount,
+  makeGetTokenPrice
+} from '../adapters'
+import { hasEIP1559Support } from '../../../../utils/network-utils'
+
+function App () {
```

Comment:
Can we simply import and re-use the component from `components/brave_wallet_ui/page/screens/swap/swap.tsx`?

Reply:
++

Reply:
Yes, fixed the android swap page to re-use component

Reply:
May be we should hide this somehow for Android?
![image](https://user-images.githubusercontent.com/118171981/220889340-26d7f71c-0dc1-4cca-943b-abe36ce46e66.png)

Reply:
Fixed